projects/sb-user-management/src/lib/user-count/user-count.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { UserCountService } from '../user-count.service';
+import tenanats from  './tenantDetails.json';
+
+interface TenantDetails {  
+  tenantName: string;  
+  tenantLogo: string;  
+  userCount: Number;  
+  channelId: string;  
+}  
+
+@Component({
+  selector: 'lib-user-count',
+  templateUrl: './user-count.component.html',
+  styleUrls: ['./user-count.component.css']
+})
+export class UserCountComponent implements OnInit {
+
+  constructor(private userCountService: UserCountService) { }
+
+  tenantDetail: TenantDetails[]=tenanats;
```

Comment:
@poojakarma 
Please correct the spelling **tenanats**. You can either user **tenantList** as the Json is referring to the list of tenants

Reply:
Done